docs/tutorials/web.md

```diff
@@ -5,10 +5,421 @@ has_children: false
+
+2. Create a model and export to ONNX Format
+
+  There are many ways to create a custom model that is specific to the task you are trying to solve. Use code to build your model or use the low code/no code tools to create the model. Check out the resources below to learn about some different ways to create a customized model. All of these resources have an export to ONNX format functionality so that you can leverage this template and source code. 
+
+  - [Use AutoML to create a custom model](https://docs.microsoft.com/azure/machine-learning/concept-automated-ml)
+  - [Use Custom Vision Cognitive Services to create a custom model](https://docs.microsoft.com/azure/cognitive-services/custom-vision-service/overview)
+  - [Use Azure Machine Learning Designer to create a custom model](https://docs.microsoft.com/en-us/azure/machine-learning/concept-designer)
+  - [Build your own model with PyTorch.](https://docs.microsoft.com/learn/paths/pytorch-fundamentals/)
+
+## Create a Static Site with NextJS (a ReactJS Framework) to deploy models in the browser
+
+### The Template
+{: .no_toc }
+The goal of this template is to provide a starting point for your accelerated ML web application. The template generates a computer vision application using the [NextJS](https://nextjs.org/) framework, authored in typescript and built with webpack. Let’s dive into the template and breakdown the code.
+
+### The `utils` Folder 
+{: .no_toc }
+There are three files in the Utils folder `imageHelper.ts`, `modelHelper.ts` and `predict.ts`. Predict is the entry point from the web component to start inferencing. Here we import the helpers and call the default functions to get the image tensor and to run our model inference. 
+
+### predict.ts
+{: .no_toc }
+```javascript
+// Language: typescript
+// Path: react-next\utils\predict.ts
+import getImageTensorFromPath from './imageHelper';
+import {runSqueezenetModel} from './modelHelper';
+
+export async function inferenceSqueezenet(path: string): Promise<[any,number]> {
+  // 1. Convert image to tensor
+  const imageTensor = await getImageTensorFromPath(path);
+  // 2. Run model
+  const [predictions, inferenceTime] = await runSqueezenetModel(imageTensor);
+  // 3. Return predictions and the amount of time it took to inference.
+  return [predictions, inferenceTime];
+}
+```
+### imageHelper.ts
+{: .no_toc }
+First, we need to get our image from path (can be local or url) and convert it to a tensor. The `getImageTensorFromPath` function in the `imageHelper.ts` uses `JIMP` to read the file, resize and return the `imageData`. [JIMP](https://www.npmjs.com/package/jimp) is a JavaScript image manipulation library. It has many built in functions for working with image data such as resizing, grey scale, write, and more. In this example we only need to resize however in your code you may need additional image data processing. 
+
+```javascript
+import Jimp  from 'jimp/*';
+import { Tensor } from 'onnxruntime-web';
```

Comment:
{Tensor} for consistency

Reply:
I chose to have a space before and after the import, I have made all the files be consistent to follow that design